### ISSUE TITLE
Where is the Linux on Z version?

### ISSUE THREAD

**Issue by mwbonett** [IBM-Bluemix-Docs/cli#63]
This page does not show a link to the Linux on Z version. Is there one? If so, it should be added.

**Comment by plindblue** [comment]
Are you referring to the ppc64le version? If so that will not be released until the latest version of the CLI 0.6.7 is released. This is undergoing some final tests and should be released soon.

**Comment by mwbonett** [comment]
No, I am referring to the s390x version. It is listed here: http://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/0.6.6/ . But the link about does not show it. In fact, at that site it shows s390x versions back to 0.6.2.

**Comment by mwbonett** [comment]
That is unfortunate, as IBM Cloud Private is being heavily promoted for use with Linux on Z, so it is unfortunate that the link the product documentation points to install the IBM Cloud CLI but there is not a Z version for Z users. I will discuss it with them to include the other link. Thanks!
